backend/events/models.py

```diff
@@ -6,8 +6,9 @@
 class Event(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField(blank=True, null="True")
-    start_time = models.DateTimeField(null="False")
-    end_time = models.DateTimeField(null="False")
+    start_date = models.DateField(blank=True, null="True")
```

Comment:
What's the logic behind making the start and end dates nullable? Also, please standardize the order of arguments in these.

Reply:
We wrote it like this for testing purposes